.travis.yml

```diff
@@ -11,7 +11,7 @@ env:
 install:
   - pip install --upgrade pip
   - pip install --progress-bar off torch==$TORCH_VER
-  - pip install --progress-bar off .[ner,srl,txtgen,stanfordnlp,test,example,indexer]
+  - pip install --progress-bar off .[ner,srl,txtgen,stanfordnlp,test,example,indexer,codegen]
```

Comment:
codegen requirements would be part of core, so we don't need to use an extra keyword here.